src/sentry/static/sentry/app/views/organizationIntegrations/integrationRepos.jsx

```diff
@@ -162,11 +193,13 @@ export default class IntegrationRepos extends AsyncComponent {
     });
 
     let menuHeader = <StyledReposLabel>{t('Repositories')}</StyledReposLabel>;
+    let onChange = this.state.integrationRepos.searchable ? this.handleSearchRepositories.bind(this) : {};
```

Comment:
Go ahead and just make `handleSearchRepositories` an instance property, no need to bind.

Reply:
`null` might be better than empty object. Does react complain at all?